### PR TITLE
add .nvmrc and setup file

### LINKED ISSUE #642: Add .nvmrc file to repo
A request from Christoph is to add a ".nvmrc" file to the repo. This allows the repo to auto-switch to an appropriate node version when installing dependencies.

### PR DESCRIPTION
**Pull request by sabakhilji**
closes #642

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Thanks for submitting this pr Saba! Can you change the name of the PR to what you have in the PR description? add .nvmrc and setup shell file

Then in the PR description you can say that it closes issue 649.

Thanks!

**Comment by sabakhilji** [comment]
> Thanks for submitting this pr Saba! Can you change the name of the PR to what you have in the PR description? add .nvmrc and setup shell file
> 
> Then in the PR description you can say that it closes issue 649.
> 
> Thanks!

Oh, I am sorry I was in a hurry :) I updated it now. Please check. Also, I need to update the readme file after someone reviewed this.

**Comment by ecurrencyhodler** [comment]
Awesome thanks for updating the name Saba!

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+16.20.0
```
Comment (benalleng) [inline .nvmrc:1]: Sorry! I was over zealous with the necessary version, the max needed is `16.13.0`

File: setup.sh
Comment (benalleng) [inline setup.sh:1]: Seems good to me, I am not completely sure this file is totally necessary for onboarding new devs but It looks good otherwise!

**Review by benalleng — CHANGES_REQUESTED** [review]
Small change for the version needed, doesn't actually need to be that high but otherwise it looks good!

File: .nvmrc (hunk 1 above)
Comment (sabakhilji) [reply .nvmrc]: done !!

File: setup.sh
Comment (sabakhilji) [reply setup.sh:1]: So should we include this?

Comment (benalleng) [reply setup.sh:1]: can't hurt!

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```
Comment (benalleng) [inline setup.sh:2]: I dont think a full addition to the README is necessary ultimately maybe just a one line comment here saying 
`#run this file if you are unsure of your installed node-version`

**Review by benalleng — APPROVED** [review]
LGMT!

File: setup.sh (hunk 2 above)
Comment (benalleng) [reply setup.sh:2]:
```suggestion

#Run this file if you are unsure of your installed node version
```

**Comment by benalleng** [comment]
Hmm this seems to have caused our prod branches to break again..

**Pull request merged**